crypto/armcap.c

```diff
@@ -354,6 +355,7 @@ void OPENSSL_cpuid_setup(void)
         if (sigsetjmp(ill_jmp, 1) == 0) {
             _armv8_sm3_probe();
             OPENSSL_armcap_P |= ARMV8_SM3;
+        }
```

Comment:
!!! How did this ever work?

Reply:
Yeah, it was clearly never even tested :(

Reply:
(which I did do before this PR)

Reply:
> Yeah, it was clearly never even tested :(

Which is why we fortunately have alpha and beta pre-releases. As we cannot test on all environments that might be possibly supported by OpenSSL code base. :grin: 

The message to anyone who is using OpenSSL on platform that is not on the primary or secondary list - test the prereleases if you want the final release to support your platform.